pkg/migrations/op_rename_constraint.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package migrations
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/lib/pq"
+	"github.com/xataio/pgroll/pkg/schema"
+)
+
+var _ Operation = (*OpRenameConstraint)(nil)
+
+func (o *OpRenameConstraint) Start(ctx context.Context, conn *sql.DB, stateSchema string, s *schema.Schema, cbs ...CallbackFn) (*schema.Table, error) {
+	table := s.GetTable(o.Table)
+	table.RenameConstraint(o.From, o.To)
```

Comment:
As described in the other comment, this can be removed and the `Start` operation can be a no-op.

Drop index is another example of a similar kind of operation with a no-op `Start`.